pydrive2/auth.py

```diff
@@ -446,6 +460,13 @@ def SaveCredentialsFile(self, credentials_file=None):
                 "Credentials file cannot be symbolic link"
             )
 
+    def SaveCredentialsDictionary(self):
```

Comment:
(I know this is still WIP), but probably unless we want to be able to provide a custom "memory" location, these two SaveCredentials and LoadCredentials are not needed as public functions and can be embedded probably.

Reply:
Great point! Will adjust.